elisp/init-haskell.el

```diff
@@ -38,6 +38,17 @@
 ;; HaskellModePac
 (use-package haskell-mode
   :mode "\\.hs\\'")
+
+;; Install and configure lsp-haskell
+(use-package lsp-haskell
+  :ensure t
+  :config
+  (setq lsp-haskell-server-path "haskell-language-server")
+  )
```

Comment:
Please put this on the same line

Reply:
Like following?
`   :config (setq lsp-haskell-server-path "haskell-language-server"))`

Reply:
Yes